wgpu-native/native/default_nsview/default_nsview.m

```diff
@@ -0,0 +1,10 @@
+#include "default_nsview.h"
+#include <AppKit/AppKit.h>
+
+void *get_default_nsview_if_null(void *ns_view_handle, void *ns_window_handle) {
```

Comment:
let's implement this in Rust, since we can (we can, right?), so that we don't have to mix languages

Reply:
I don't think that's going to be possible because I have to call an Objective-C function: https://github.com/gfx-rs/wgpu/pull/462/files#diff-987af146394a41522a5ae1931a5592acR9

Reply:
You'd add a macos-only dependency for `objc` crate, which allows you to send Obj C message. We use plenty of these at lower level, in gfx-backend-metal: https://github.com/gfx-rs/gfx/blob/f999b5295ca30dc38f33b0a4770a31d50138687c/src/backend/metal/src/lib.rs#L369

Reply:
No way! Ok let me add that in real quick.

Reply:
I've made those changes. Let me know how it looks.